stl/inc/xutility

```diff
@@ -4352,10 +4352,11 @@ template <>
 struct _Is_character_or_byte_or_bool<bool> : true_type {};
 
 // _Fill_memset_is_safe determines if _FwdIt and _Ty are eligible for memset optimization in fill
+// Need to explicity test for volatile because _Unwrap_enum_t discards qualifiers.

```

Comment:
```suggestion
// _Fill_memset_is_safe determines if _FwdIt and _Ty are eligible for memset optimization in fill.
// Need to explicitly test for volatile because _Unwrap_enum_t discards qualifiers.
```